k2/util/Logging.cc

```diff
@@ -0,0 +1,57 @@
+// $FILE_PATHNAME $HEADER_FILENAME $HEADER_PATHNAME
+
+// Copyright (c) 2020 Xiaomi Corporation ( authors: Meixu Song )
+//
+
+// See ../../LICENSE for clarification regarding multiple authors
+
+#include "k2/util/Logging.h"
+
+#ifdef K2_USE_GLOG
+// Google glog's api does not have an external function that allows one to check
+// if glog is initialized or not. It does have an internal function - so we are
+// declaring it here. This is a hack but has been used by a bunch of others too.
```

Comment:
RE `has been used by a bunch of others`, it's better if we could list some famous project here with the corresponding code links.

Reply:
Done